### PR TITLE
Sinead Coughlan's QuickStart topic

### PR DESCRIPTION
**Pull request by sinecoug**
Proposing a commit that comprises - 

1. My QuickStart topic (my_first_use_topic_sineadc.md)
2. Updated index.md with link to #1.

### CODE REVIEW

File: docs/api/quickstart_sinead.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+* **A host.**  The {base_url} depends on users' installation of the service in their development environment. For v1 of To-Do Service API, the **base_url** variable is typically set to `http://localhost:3000`.
+* **Authorization.**  For v1 of the To-Do service, requests do not use any authorization. All endpoints are available to all users and applications.
+* **A request.**  The To-Do service REST API enables CRUD operations via HTTP requests on database resources (`GET`, `POST`, `PUT`, `PATCH`, and `DELETE` methods). Request and response bodies are encoded as JSON.
+
+### Supported endpoints
+
+| HTTP Method | Endpoint |
+| :--------------: | :--------------: |
+| GET | [List all users](https://uwc2-apidoc.github.io/to-do-service-public/api/users-get-all-users) |
```
Comment (rbwatson) [inline docs/api/quickstart_sinead.md]:
```suggestion
| GET | [List all users](users-get-all-users.md) |
```
Is there a reason you used the complete URL in the links? Normally, for topics in the documentation set, a relative link (such as the suggestion) is preferred for a variety of reasons, such as: it's shorter, the IDE can validate it's existence, the web server can optimize its delivery.

If a relative link will work, can you update the link targets, please?
Please update the link

File: docs/api/quickstart_sinead.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+| GET | [List user by ID](https://uwc2-apidoc.github.io/to-do-service-public/api/users-get-user-by-id) |
+| GET | [List user by email](https://uwc2-apidoc.github.io/to-do-service-public/api/users-get-user-by-email) |
+| POST | [Create a user](https://uwc2-apidoc.github.io/to-do-service-public/api/users-create-user) |
+| PUT | [Update user by ID](https://uwc2-apidoc.github.io/to-do-service-public/api/users-update-by-id) |
+| PATCH | [Update user email](https://uwc2-apidoc.github.io/to-do-service-public/api/users-change-user-email.html) |
+| PATCH | [Update user properties](https://uwc2-apidoc.github.io/to-do-service-public/api/users-change-user-property) |
+| DELETE | [Delete user by email](https://uwc2-apidoc.github.io/to-do-service-public/api/users-delete-user-by-email) |
```
Comment (rbwatson) [inline docs/api/quickstart_sinead.md]:
```suggestion
```
This page is no longer in the documentation (the feature isn't supported).

**Review by rbwatson — CHANGES_REQUESTED** [review]
This is a great document overall.
Please update the link targets per the comments and it'll be ready to merge.

**Comment by sinecoug** [comment]
Thanks, including absolute links (html) was an oversight on my part! I've updated the file with relative links.

**Review by rbwatson — APPROVED** [review]
Rebased and squashed for merge.
Passed link test.
LGTM

**Pull request merged**